server/app/api/controllers/projects.js

```diff
@@ -57,11 +57,9 @@ module.exports = {
       }
     });
   },
-  getAll: function(req, res, next) {    
-    //todo - remove or optimize it
-    const status = req.query.status
-    const query = status ? {status} : {}
-    projectModel.find(query, function(err, projects) {
+  getAll: function(req, res, next) { 
+    let {status} = req.query;  
```

Comment:
why let? status value is not changing...

Reply:
Missed . Updated now.